src/Microsoft.DotNet.Interactive.Http.Tests/ParserTests.Variables.cs

```diff
@@ -250,5 +250,21 @@ public void underscores_in_embedded_expressions_are_supported()
             variables.Should().Contain(n => n.Key == "host").Which.Value.Should().BeOfType<DeclaredVariable>().Which.Value.Should().Be("https://httpbin.org");
 
         }
+
+        [Fact]
+        public void spaces_after_variable_do_not_produce_diagnostics()
+        {
+            var result = Parse(
+                """
+                @host=https://httpbin.org
+                
+                
+                
+                
+                
+                """);
+
+            result.SyntaxTree.RootNode.ChildNodes.Count().Should().Be(1);
```

Comment:
This assertion can be more specific.  It doesn't currently seem related to the test name.

Reply:
Done